uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/apis/leave-request-api.js

```diff
@@ -240,6 +241,67 @@ define([
           .then(function (response) {
             return response.values;
           });
+      },
+
+      /**
+       * Calls the getcomment backend API.
+       *
+       * @param {String} leaveRequestID - Id of leave request
+       * @param {Object} params
+       *
+       * @return {Promise}
+       */
+      getComments: function (leaveRequestID, params) {
+        params = _.assign(params, {
```

Comment:
if params is null then it will make it empty, better to start with empty object in assign 
`params = _.assign({}, params`